pkg/services/accesscontrol/dualwrite/collectors.go

```diff
@@ -397,7 +414,8 @@ func zanzanaCollector(relations []string) zanzanaTupleCollector {
 
 			for c != "" {
 				res, err := client.Read(ctx, &authzextv1.ReadRequest{
-					Namespace: namespace,
+					ContinuationToken: c,
```

Comment:
There was a bug caused an endless loop here due to lack of continuation token provided

Reply:
Oh, that must have happen when we wrapped read :thinking: